### PR TITLE
Steps component bubbles aliasing

### PR DESCRIPTION
**Pull request by leonardohofling**
Resolve opencollective/opencollective#3716

# Description

Steps component bubbles aliasing

# Screenshots

image

### CODE REVIEW

**Comment by leonardohofling** [comment]
@Betree Could you take a look, please? I tested using Linux Mint but with a newer version of chrome.

**Review by Betree — COMMENTED** [review]
That definitely resolves the aliasing issue, great!

We need to make sure it stays consistent with the design, there are minor regressions. Taking https://www.figma.com/file/N4Xbl652BhzOutXmzhcrLGch/%5BDS%5D-02-Atoms?node-id=14%3A3 as a reference:

- Bubble's border width is too thick when not active. According to the design, it should be 1px. Color also doesn't match, should be `#C4C7CC`
- Font size of the number also doesn't match design

**Comment by leonardohofling** [comment]
> That definitely resolves the aliasing issue, great!
> 
> We need to make sure it stays consistent with the design, there are minor regressions. Taking https://www.figma.com/file/N4Xbl652BhzOutXmzhcrLGch/%5BDS%5D-02-Atoms?node-id=14%3A3 as a reference:
> 
> * Bubble's border width is too thick when not active. According to the design, it should be 1px. Color also doesn't match, should be `#C4C7CC`
> * Font size of the number also doesn't match design

@Betree changes done =)

File: components/StepsProgress.js [hunk 1]
```diff
@@ -202,17 +245,20 @@ const PieHalfCircleRight = styled(PieHalfCircle)`
...
     return <Check color="white" size={14} />;
   }
 
   return (
-    <Span fontWeight={400} fontSize="14px" lineHeight="20px">
-      {idx + 1}
-    </Span>
+    <Circle disabled={disabled} checked={checked} focus={focused}>
+      <circle cx="50%" cy="50%" r="16px"></circle>
+      <text x="50%" y="50%" dominantBaseline="middle" textAnchor="middle">
```
Comment (Betree) [inline components/StepsProgress.js:258]: I'm not sure why, but with `50%` the text appear a little too high. `51%` seems to resolve the issue.

image

```suggestion
      <text x="50%" y="51%" dominantBaseline="middle" textAnchor="middle">
```

**Review by Betree — APPROVED** [review]
Looking great, thank you @leonardohofling!

**Pull request merged**